accounts/templates/accounts/profile_settings.html

```diff
@@ -0,0 +1,73 @@
+{% extends 'landing_page/base.html' %} {% load static %} {% block content %}
+
+<link rel="stylesheet" href="{% static 'accounts/css/settings.css' %}" />
+<link rel="stylesheet" href="{% static 'accounts/css/modal.css' %}" />
+
+<div id="message_div">
+  <h1>Account Settings</h1>
+</div>
+<div class="content_div">
+  <a href="{% url 'landing_page:profile' %}">
+    <button id="settings_btn">Back</button>
+  </a>
+  <div class="settings_container" id="profile_info">
+    <div class="left_col">
+      <h2>Profile</h2>
+      <p>Edit your Profile picture, Username, and Password.</p>
+    </div>
+    <div class="right_col">
+      <img src="{% static 'landing_page/images/profile.png' %}" />
+    </div>
+  </div>
+  <div class="settings_container" id="data_imports">
+    <div class="left_col">
+      <h2>Imported Data</h2>
+      <p>Import or remove data from IMDB, Letterboxd, and Netflix.</p>
+    </div>
+    <div class="right_col">
+      <button>Import Data</button>
+    </div>
+  </div>
+  <div class="settings_container" id="movie_preferences">
+    <div class="left_col">
+      <h2>Preferences</h2>
+      <p>Edit your prefered genres, triggers, etc.</p>
+    </div>
+    <div class="right_col">
+      <button>Edit Preferences</button>
+    </div>
+  </div>
+  <div class="settings_container" id="delete_account">
+    <div class="left_col">
+      <h2>Delete Account</h2>
+      <p>
+        Upon deletion your account and it's information will be permanently
```

Comment:
little clunky. maybe more like 
"Your account and its information will be permanently deleted. This action cannot be undone."